app/views/shared/_available_languages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if non_english_translations(translations).any? %>
```

Comment:
Not sure this is correct. There are times when we might have content which is translated into more than one language and neither of them will be english. I think this really just wants to check for more than one translation.